.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
Does this need to be committed?

Reply:
No, it was created automatically. I'll just delete it from github.